packages/xchain-cosmos/src/util.ts

```diff
@@ -65,31 +74,30 @@ export const getAsset = (denom: string): Asset | null => {
  */
 export const getTxsFromHistory = (txs: TxResponse[], mainAsset: Asset): Tx[] => {
   return txs.reduce((acc, tx) => {
-    let msgs: Msg[] = []
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let msgs: any[]
```

Comment:
Why `any` - does `MsgSendMsgMultiSend` not work? You can use type guards `isMsgSend` / `isMsgMultiSend` then.

Reply:
fixed